tools/gulptasks/dashboards/scripts-watch.js

```diff
@@ -83,24 +80,12 @@ async function task() {
 
     processLib.isRunning('dashboards/scripts-watch', true);
 
-    if (argv.dts) {
-        await gulp.task('jsdoc-dts')();
-    }
-
     return processLib
```

Comment:
If you label the task function `async`, then you just need an `await` instead of `return` and `then`.